docs/getting-started.md

```diff
@@ -73,6 +73,36 @@ Antrea supports some experimental features that can be enabled or disabled,
 please refer to the [Feature Gates documentation](feature-gates.md) for more
 information.
 
+### Antrea ARM support
```

Comment:
What you think to create a separate doc for ARM and put a link here? Want not to make getting-started too long.

Reply:
no strong preference on my side. It should be a pretty temporary section and I'm fine moving it to its own file. Will update the PR.